app/helpers/admin/site_maps_helper.rb

```diff
@@ -3,12 +3,26 @@
 module Admin::SiteMapsHelper
   include Admin::AdminHelper
 
+  def sitemap_pages
+    @sitemap_pages ||= Page.where(:_type.in => %w(Page SetPage)).entries
```

Comment:
Btw instead of `.entries`, you could do `group_by(&:page_id)`. Then you end up with a hash mapping page ids to page children. Then `sitemap_children_for` should be faster because you're just doing a hash lookup rather than a `select`.